src/init-ceph.in

```diff
@@ -740,6 +740,25 @@ if [ "$command" = "stop" -o "$command" = "onestop" ]; then
     what="$new_order"
 fi
 
+# Check if the monitors are up before starting any mds
+what_out=
+. /etc/platform/platform.conf
+
+if [ "$system_type" == "Standard" ];then
+        timeout 10 ceph -s
+        if [ "$?" -ne 0 ]; then
+        for name in $what;
+        do
+                type=`echo $name | cut -c 1-3`
```

Comment:
you can replace it with:
if [[ $name =~ ^mds\. ]]; then

format is e.g. mds.controller-0, isn't it?

Reply:
yes, but I was trying tot keep style consistent with the other changes